app/assets/stylesheets/home.scss

```diff
@@ -0,0 +1,73 @@
+
+$explore-background: #449662;
+$box-background: #abe3c0;
+$box-footer-background: #4d785d;
+.home {
+  .left-column {
+    @include span-columns(10);
+    margin-top: 20px;
+  }
+  .right-column {
+    @include span-columns(2);
+    margin-top: 20px;
+
+    h2 {
+      @include padding(20px);
+    }
+
+    li {
+      @include padding(20px);
+      border-bottom: 1px solid #EBECE7;
+    }
+  }
+
+  .lastest-books {
+    @include display(flex);
+    @include justify-content(space-around);
+    li {
+      z-index: 1;
+    }
+  }
+
+  .box {
+    @include border-left-radius(15px);
+    @include border-right-radius(15px);
+    position: relative;
+    background-color: $box-background;
+    .box-title {
+      padding: 20px;
+      font-style: italic;
+    }
+
+  }
+
+  .box-footer {
+    display: block;
+    background-color: $box-footer-background;
+    height: 30px;
+    position: absolute;
+    bottom: 1px;
+    width: 100%;
+  }
+}
+
+.explore {
+  @include padding(40px 0 40px 0);
+  background-color: $explore-background;
+  color: white;
+
+  button {
+    @include button(simple, white);
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.